src/schemas/686c-674/schema.js

```diff
@@ -478,12 +478,13 @@ const schema = {
         location: {
           $ref: '#/definitions/genericLocation',
         },
-        isMarriageAnnulledOrVoid: {
-          $ref: '#/definitions/genericTrueFalse',
-        },
-        explanationOfAnnullmentOrVoid: {
+        reasonMarriageEnded: {
```

Comment:
Renamed this property and the one below it, since we're no longer explicitly asking if the marriage was annulled or declared void.